triggers/daml/Daml/Trigger.daml

```diff
@@ -104,6 +103,7 @@ emitCommands cmds pending = do
     , pendingContracts = Map.insert id pending s.pendingContracts
     , nextCommandId = s.nextCommandId + 1
     }
+  submitCommands commands
```

Comment:
Not part of this PR (I might have already mentioned this in #7392 not quite sure) but we could consider deprecating the name `emitCommands` in favor of `submitCommands`. `emit` isn’t really terminology that we use anywhere else and just seems confusing. I don’t quite want to use just `submit` since we use that for the synchronous submissions in scenarios and daml script.

Reply:
`submitCommands` is the low-level primitive corresponding to "add an element to the returned `[Commands]`". As far as I can tell, the distinction with `emitCommands` needs to be preserved. The names can be anything.

Reply:
Right I wasn’t suggesting to remove the distinction, just cleaning up names for the high-level API. But definitely not something for this PR and maybe sticking to `emit` is also sensible. I didn’t spend too much time thinking about this.